usr/sbin/rear

```diff
@@ -50,23 +68,34 @@ function apply_bash_flags_and_options_commands () {
 #   apply_bash_flags_and_options_commands "$INITIAL_BASH_FLAGS_AND_OPTIONS_COMMANDS"
 readonly INITIAL_BASH_FLAGS_AND_OPTIONS_COMMANDS="$( get_bash_flags_and_options_commands )"
 
-# Enable as needed for development of fail-safe code:
-# set -ue -o pipefail
-# set -xue -o pipefail
-# set -xvue -o pipefail
-# Cf. the Relax-and-Recover Coding Style
-# https://github.com/rear/rear/wiki/Coding-Style
-# that reads (excerpt - dated 18. Nov. 2015):
-# "TODO Use set -eu to die from unset variables and unhandled errors"
-
-# Versioning
-readonly PRODUCT="Relax-and-Recover"
-readonly PROGRAM=${0##*/}
-readonly VERSION=2.9
-readonly RELEASE_DATE="2025-01-31"
-
-# Where users should report bugs:
-readonly BUG_REPORT_SITE="https://github.com/rear/rear/issues"
+# To be on the safe side forbid careless/insecure sourcing
+# via plain 'source' and '.' from the very beginning
+# see in general https://relax-and-recover.org/documentation/security-architecture
+# and see in particular https://github.com/rear/rear/issues/3259
+# and https://github.com/rear/rear/issues/3259#issuecomment-2385745545
+# and https://github.com/rear/rear/issues/3319#issuecomment-2360546784
+# and https://github.com/rear/rear/issues/3319#issuecomment-2363556217
+function . () {
+    echo -e "ERROR: BUG in $PRODUCT\nForbidden usage of '.' in\n  . $*\nPlease report it at $BUG_REPORT_SITE" >&2
```

Comment:
```suggestion
    echo -e "ERROR: BUG in $PRODUCT\nForbidden usage of Bash '.' instead of ReaR 'Source' in\n  . $*\nPlease report it at $BUG_REPORT_SITE" >&2
```

Reply:
It is not about using ReaR's 'Source' function
because this function is only to source ReaR files
(ReaR scripts or ReaR config files).

I will change the text to
```
ERROR: BUG in $PRODUCT\nForbidden usage of '.' instead of 'source' in\n  . $*\nPlease report it at $BUG_REPORT_SITE
```